PlaceRouteHierFlow/PnR-pybind11.cpp

```diff
@@ -491,8 +491,10 @@ PYBIND11_MODULE(PnR, m) {
     .def_readwrite("use_external_placement_info", &PlacerHyperparameters::use_external_placement_info)
     .def_readwrite("max_init_trial_count", &PlacerHyperparameters::max_init_trial_count)
     .def_readwrite("max_cache_hit_count", &PlacerHyperparameters::max_cache_hit_count)
-    .def_readwrite("select_in_ILP", &PlacerHyperparameters::select_in_ILP)
-    .def_readwrite("ilp_solver", &PlacerHyperparameters::ilp_solver)
+    .def_readwrite("select_in_ILP", &PlacerHyperparameters::select_in_ILP) // select variant of blocks in SA using ILP used for compaction
```

Comment:
Does this mean?
select_in_ILP == False => Variants will be chosen in sequence pair representation
select_in_ILP == True => Variants will be chosen by ILP solver?

Reply:
@soneryaldiz Yes. This was added by @854768750 upon Steve's proposal sometime ago.

Reply:
The old version with lpsolve uses this.

Reply:
@854768750 , could you please elaborate? What do you mean by old version? Is this switch no longer functional?

Reply:
We used to have tests for this but it was so slow we commented it out.

Reply:
The number of decision variables may be several times more if variants a are chosen by ILP solver